ops/pebble.py

```diff
@@ -1407,12 +1427,21 @@ class CheckInfo:
     This can be :attr:`CheckLevel.ALIVE`, :attr:`CheckLevel.READY`, or None (level not set).
     """
 
+    startup: Union[CheckStartup, str]
```

Comment:
I think the type here should definitely be just `CheckStartup`, right, as it's always going to be set when returned from Go in the check info?

Oh, I guess there's the case where it's not present (older versions of Pebble) -- but in that case, it's ENABLED, right?